diff --git a/node_modules/react-native/gradle/libs.versions.toml b/node_modules/react-native/gradle/libs.versions.toml
index 43c8bb3..3bf0c92 100644
--- a/node_modules/react-native/gradle/libs.versions.toml
+++ b/node_modules/react-native/gradle/libs.versions.toml
@@ -2,11 +2,11 @@
 # Android versions
 minSdk = "24"
 targetSdk = "35"
-compileSdk = "35"
-buildTools = "35.0.0"
+compileSdk = "36"
+buildTools = "36.0.0"
 ndkVersion = "27.1.12297006"
 # Dependencies versions
-agp = "8.8.2"
+agp = "8.9.2"
 androidx-annotation = "1.6.0"
 androidx-appcompat = "1.7.0"
 androidx-autofill = "1.1.0"
